antora-playbook.yml

```
@@ -19,4 +19,8 @@ asciidoc:
     linkattrs: ''
     toc: ~
     quarkus_version: '2.9.1.Final'
-    required_maven_version: '3.8.1+'
\ No newline at end of file
+    required_maven_version: '3.8.1+'
+    java_min_version: 11+
+    maven_min_version: 3.8.1+
+    graalvm_min_version: 22.0.0.2
+    kogito-examples: https://github.com/kiegroup/kogito-examples.git
```

```suggestion
    kogito_examples_url: https://github.com/kiegroup/kogito-examples.git
```
Sorry for the nit, but we will use this format as the convention. To be written in the CONTRIB. 😓